opentelemetry-sdk/src/metrics/mod.rs

```diff
@@ -209,4 +209,57 @@ mod tests {
         let datapoint = &sum.data_points[0];
         assert_eq!(datapoint.value, 15);
     }
+
+    // "multi_thread" tokio flavor must be used else flush won't
+    // be able to make progress!
+    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+    async fn histogram_aggregation_with_invalid_aggregation_should_proceed_as_if_view_not_exist() {
+        // Run this test with stdout enabled to see output.
+        // cargo test histogram_aggregation_with_invalid_aggregation_should_proceed_as_if_view_not_exist --features=metrics,testing -- --nocapture
+
+        // Arrange
+        let exporter = InMemoryMetricsExporter::default();
+        let reader = PeriodicReader::builder(exporter.clone(), runtime::Tokio).build();
+        let criteria = Instrument::new().name("test_histogram");
+        let stream_invalid_aggregation = Stream::new()
+            .aggregation(Aggregation::ExplicitBucketHistogram {
+                boundaries: vec![0.9, 1.9, 1.2, 1.3, 1.4, 1.5], // invalid boundaries
+                record_min_max: false,
+            })
+            .name("test_histogram_renamed")
+            .unit(Unit::new("test_unit_renamed"));
+
+        let view =
+            new_view(criteria, stream_invalid_aggregation).expect("Expected to create a new view");
```

Comment:
Wondering how/where would we validate the view config, when one is using https://github.com/open-telemetry/opentelemetry-rust/blob/main/opentelemetry-sdk/src/metrics/view.rs#L56-L63 instead? Might require us to move the validation elsewhere....

Reply:
https://github.com/open-telemetry/opentelemetry-rust/issues/1414 Opened this issue to track doing a proper fix.